src/index.js

```diff
@@ -1,2 +1,58 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
+
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render(state = { count: 0 }) {
```

Comment:
```suggestion
function render({ count = 0}) {
```

기본 값을 넣고 싶으시면 위와 같이 표현하면 편할 겁니다. 굳이 state하는 이름을 명시하기 보단

어차피 던져준 객체를 바로 구조분해 할당 하는게 더 간결하죠.

그리고 저라면 그냥 render({ count }) 로 단순하게 유지할 것 같아요.

처음 58라인에서 호출할 때 render({ count: 0 }); 를 하면 되니까요.